recipe/meta.yaml

```
@@ -22,7 +22,7 @@ requirements:
     - pip
     - python >=3.6
   run:
-    - ipywidgets >=7.6.0,<8.0.0
+    - ipywidgets >=7.6.0
```

```suggestion
    - ipywidgets >=7.6.0,<9.0.0
```

probably a good idea to pin below v9

---

I'm not sure how safe that is, because of the unpinned releases. If you do `conda create -p /tmp/rfb 'ipywidgets>=8' jupyter-rfb`, then it installs 0.3.1, even though that version combination is broken. Is there a way to block/replace 0.3.1 with a pin, so that this pin would be effective?

---

are you basically trying to get this with ipywidgets v8? I'd wait for #6 , I'll take a look too if I can get access to retry the build

---

AFAICT, 0.3.3 should be okay with ipywidgets v8. And the failure in #6 would be fixed by the second commit here.